apps/key-manager/src/components/auth/SignOut.tsx

```diff
@@ -0,0 +1,21 @@
+"use client";
+
+import { DropdownMenuItem } from "@/components/ui/dropdown-menu";
+import { LogOutIcon } from "lucide-react";
+import { signOut } from "next-auth/react";
+
+export default function SingOut() {
```

Comment:
Component still misnamed.

Reply:
`export default` allowed the correct name to be used on import despite this typo. This line is the last typo of this type as far as I know.